src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue

```diff
@@ -66,6 +95,12 @@ export default {
 				});
 
 				if (activePromoBanner) {
+					// Check banner type
+					if (activePromoBanner.fields.bannerType === 'Appeal Banner') {
+						this.appealEnabled = true;
+						this.appealBannerContent = activePromoBanner;
+					}
```

Comment:
This whole block should move to the bottom and have an `else` statement that now contains `this.isPromoEnabled = true;` You could also move the assignment of `this.promoBannerContent = {...` into the else block before setting isPromoEnabled to true.